modules/installing-ove-booting.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * virt/install/installing-booting.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-installing-ove-booting_{context}"]
+= Mounting the ISO and booting the machines
+
+To initiate the cluster installation, attach the downloaded ISO to each machine and boot the machines from the ISO.
+
+.Prerequisites
+
+* You have downloaded the installation ISO.
+
+.Procedure
+
+. For each machine in your environment, attach the ISO image to the machine and boot the machine from this image.
+
+. Wait for each machine to boot from the image and display the *Rendezvous node setup* menu.
+
+. Configure the rendezvous node for the cluster installation:
+
+.. On the machine that you want to run as the bootstrap host for the installation process, select *This is the rendezvous node* in the *Rendezvous node setup* menu.
```

Comment:
An important note: only *one* machine could act as a rendenzvous node. Selecting two or more instances as a rendezvous node is not correct or supported

Reply:
I can add a warning in this step to only set one machine as the rendezvous node